includes/class-wcpdf-settings-debug.php

```diff
@@ -161,136 +161,148 @@ public function init_settings() {
-				'callback'		=> 'checkbox',
-				'section'		=> 'debug_settings',
-				'args'			=> array(
-					'option_name'	=> $option_name,
-					'id'			=> 'legacy_textdomain',
-					'description'	=> __( "Legacy textdomain fallback ensures compatibility with translation files from versions prior to 2017-05-15.", 'woocommerce-pdf-invoices-packing-slips' ),
+				'type'     => 'setting',
+				'id'       => 'legacy_textdomain',
+				'title'    => __( 'Legacy textdomain fallback', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback' => 'checkbox',
+				'section'  => 'debug_settings',
+				'args'     => array(
+					'option_name' => $option_name,
+					'id'          => 'legacy_textdomain',
+					'description' => __( "Legacy textdomain fallback ensures compatibility with translation files from versions prior to 2017-05-15.", 'woocommerce-pdf-invoices-packing-slips' ),
 				)
 			),
 			array(
-				'type'			=> 'setting',
-				'id'			=> 'guest_access',
-				'title'			=> __( 'Allow guest access', 'woocommerce-pdf-invoices-packing-slips' ),
-				'callback'		=> 'checkbox',
-				'section'		=> 'debug_settings',
-				'args'			=> array(
-					'option_name'		=> $option_name,
-					'id'				=> 'guest_access',
-					'description'		=> __( 'Enable this to allow customers that purchase without an account to access their PDF with a unique key', 'woocommerce-pdf-invoices-packing-slips' ),
+				'type'     => 'setting',
+				'id'       => 'guest_access',
+				'title'    => __( 'Allow guest access', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback' => 'checkbox',
+				'section'  => 'debug_settings',
+				'args'     => array(
+					'option_name' => $option_name,
+					'id'          => 'guest_access',
+					'description' => __( 'Enable this to allow customers that purchase without an account to access their PDF with a unique key', 'woocommerce-pdf-invoices-packing-slips' ),
 				)
 			),
 			array(
-				'type'			=> 'setting',
-				'id'			=> 'calculate_document_numbers',
-				'title'			=> __( 'Calculate document numbers (slow)', 'woocommerce-pdf-invoices-packing-slips' ),
-				'callback'		=> 'checkbox',
-				'section'		=> 'debug_settings',
-				'args'			=> array(
-					'option_name'	=> $option_name,
-					'id'			=> 'calculate_document_numbers',
-					'description'	=> __( "Document numbers (such as invoice numbers) are generated using AUTO_INCREMENT by default. Use this setting if your database auto increments with more than 1.", 'woocommerce-pdf-invoices-packing-slips' ),
+				'type'     => 'setting',
+				'id'       => 'pretty_document_links',
+				'title'    => __( 'Pretty document links', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback' => 'checkbox',
+				'section'  => 'debug_settings',
+				'args'     => array(
+					'option_name' => $option_name,
+					'id'          => 'pretty_document_links',
+					'description' => __( 'Changes the document links to a prettier URL scheme.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
_I'd add an example after the descriptions:_
"Changes the document links to a prettier URL scheme with the following pattern: example.com/wcpdf/{$document_type}/{$order_id}/{$nonce||$order_id}."

The home URL and the identifier would be generated to match with the actual ones. Later, we can update this description when we have written a Docs article for this setting.

Reply:
@Spreeuw what's your opinion?

Reply:
@YordanSoares I think the feature is good for now, we could improve it later if necessary, let's ship it and improve over time.

Reply:
@alexmigf I agree! 👍